server/jetty-app/build.gradle

```diff
@@ -41,6 +41,14 @@ if (!hasProperty('excludeHotspotImpl')) {
     extraJvmArgs += ['--add-opens', 'java.management/sun.management=ALL-UNNAMED']
 }
 
+if (!hasProperty('excludeClockImpl')) {
+    dependencies {
+        runtimeOnly project(':clock-impl')
+    }
+    extraJvmArgs += ['--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED']
+    extraJvmArgs += ['-DRealTimeClockLoader.timeSource=jdk-internals']
```

Comment:
again, this should be implicit based on the classpath for the majority of use-cases.